samples/django_sample/settings.py

```diff
@@ -66,12 +66,12 @@
 
 ROOT_URLCONF = 'django_sample.urls'
 
-TEMPLATE_DIRS = (
+TEMPLATE_DIRS = [
     # Put strings here, like "/home/html/django_templates"
     # Always use forward slashes, even on Windows.
     # Don't forget to use absolute paths, not relative paths.
     os.path.join(os.path.dirname(__file__), 'templates')
```

Comment:
No, there would still be Type error. It expects a list.

On Sat, Jun 18, 2016 at 11:03 PM, Nathaniel Manista <
notifications@github.com> wrote:

> In samples/django_sample/settings.py
> <https://github.com/google/google-api-python-client/pull/239#discussion_r67603967>
> :
>
> > @@ -66,12 +66,12 @@
> >
> >  ROOT_URLCONF = 'django_sample.urls'
> >
> > -TEMPLATE_DIRS = (
> > +TEMPLATE_DIRS = [
> >      # Put strings here, like "/home/html/django_templates"
> >      # Always use forward slashes, even on Windows.
> >      # Don't forget to use absolute paths, not relative paths.
> >      os.path.join(os.path.dirname(__file__), 'templates')
>
> Rather than change TEMPLATE_DIRS to a list, shouldn't a comma simply be
> added to the end of its one element?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/google/google-api-python-client/pull/239/files/83c8a7abaab31f878eaa84d25e5c6fd224cbc63b#r67603967>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AFIkWk9ewCHWh95ZvfRPMULLGCZuk8slks5qNCvbgaJpZM4I2fc5>
> .
>